remote-workstations/new-agent-vm/Install-PCoIPAgent.sh

```diff
@@ -391,22 +418,47 @@ install_pcoip_agent()
     sudo rpm -Uvh --quiet https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 
     # Install the PCoIP Agent
-    echo "-->Install the PCoIP $AGENT_TYPE agent" | tee -a "$INST_LOG_FILE"
+    local dispName=("first" "second" "third")
     for idx in {1..3}
     do
+        echo "-->Install the PCoIP $AGENT_TYPE agent at ${dispName[idx -1]} time" | tee -a "$INST_LOG_FILE"
         sudo yum -y install "pcoip-agent-$AGENT_TYPE"
         exitCode=$?
 
+        if [[ $exitCode -eq 0 ]]
```

Comment:
So the agent installer does not set firewall?

Reply:
it does set, and it works before. But now the firewalld service itself is stopped at this stage which cause installer set firewall failed. (I do not know why). So restart firewall service and set firewall after installation is done which solve the issue. This issue is occurred since last week

Reply:
Okay. Cool. Maybe add some comment about 'pcoip agent installer usually does this but it can fail if the firewall service is disabled, so re-doing here.' since otherwise it would looks strange and duplicated to a future reviewer.

Reply:
would enabling the firewall before running install make this easier?

Reply:
At the very beginning the firewall is running. so do check and enable. I will at comment like bill suggestion